torch_glow/tests/functionality/quantized_cut_in_the_middle.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+from tests.utils import GLOW_NODE_NAME, SUBGRAPH_ATTR
+import torch_glow
+import unittest
+
+
+class TestBlackList(unittest.TestCase):
+    def test_quantized_cut(self):
+        """Test cut quantized chunk in the middle."""
+        torch._C._jit_set_profiling_executor(False)
+        torch._C._jit_set_profiling_mode(False)
+
+        def fun(a, b, c, d):
+            q1 = torch.nn.quantized.Quantize(scale=1.0/128, zero_point=0,
+                                    dtype=torch.quint8)
+            q2 = torch.nn.quantized.Quantize(scale=1.0/129, zero_point=3,
+                                    dtype=torch.quint8)
+            dq = torch.nn.quantized.DeQuantize()
+            a = q1(a)
+            b = q1(b)
+            c = q2(c)
+            d = q2(d)
+            adds = torch.ops.quantized.add(a, b, scale=1.0/121, zero_point=5)
+            adds2 = torch.ops.quantized.add(c, d, scale=1.0/122, zero_point=4)
+            res = torch.ops.quantized.add_relu(adds, adds2, scale=1.0/120, zero_point=6)
+            res = torch.ops.quantized.add(res, res, scale=1.0/128, zero_point=7)
+            res = dq(res)
+            return res
+
+        with torch.no_grap():
+            a = torch.randn([5,5])
+            b = torch.randn([5,5])
+            c = torch.randn([5,5])
+            d = torch.randn([5,5])
+            res_torch = fun(a,b,c,d)
+            torch_glow.enableFusionPass()
+            # Cut using blacklist functionality
+            torch_glow.setFusionBlacklist(["quantized::add_relu"])
```

Comment:
please test that the graph is cut by asserting which nodes are in the subgraph

Reply:
Okay. I verified this locally, but you are right we need to make sure that the add_relu is excluded.